arkouda/series.py

```diff
@@ -546,31 +574,34 @@ def concat(arrays, axis=0, index_labels=None, value_labels=None):
 
                 data = idx.to_dict(index_labels)
 
-                for col, label in zip(arrays, value_labels):
-                    data[str(label)] = lookup(col.index.index, col.values, idx.index, fillvalue=0)
+                if value_labels is not None:
+                    # Expect value_labels to always be not None; were doing the check for mypy
+                    for col, label in zip(arrays, value_labels):
+                        data[str(label)] = lookup(col.index.index, col.values, idx.index, fillvalue=0)
 
-            retval = DataFrame(data)
+            return arkouda.dataframe.DataFrame(data)
         else:
             # Verticle concat
             idx = arrays[0].index
             v = arrays[0].values
             for other in arrays[1:]:
                 idx = idx.concat(other.index)
                 v = concatenate([v, other.values], ordered=True)
-            retval = Series(index=idx.index, data=v)
-
-        return retval
+            return Series(index=idx.index, data=v)
 
     @staticmethod
-    def pdconcat(arrays, axis=0, labels=None):
+    @typechecked
+    def pdconcat(
+        arrays, axis: int = 0, labels: Strings = None
+    ) -> Union[arkouda.dataframe.DataFrame, Series, pd.Series, pd.DataFrame]:
```

Comment:
we missing type annotations on `arrays`. Also this should only return `pd.Series` and `pd.DataFrame`